packages/kbn-unsaved-changes-badge/kibana.jsonc

```diff
@@ -1,5 +1,9 @@
 {
   "type": "shared-common",
   "id": "@kbn/unsaved-changes-badge",
-  "owner": "@elastic/kibana-data-discovery"
-}
+  "owner": [
+    "@elastic/kibana-data-discovery"
+  ],
+  "group": "platform",
+  "visibility": "private"
```

Comment:
This currently isn't used outside of platform, but it could be. If Solutions wanted to use it, would we just change `visibility` to `shared`?

Reply:
Yes, that's one option.

As a matter of fact, I'm mass-categorising packages according to their "dependants", following the minimum-visibility principle: If something is not used outside of platform => make it private.

But the best possible categorisation differs for each package. In this particular case, I agree that it'd probably make sense to make it `shared` instead. And nothing prevents us from moving it to `shared` in the future, only that it will be a bit more tedious once packages and plugins are relocated to [dedicated folders](https://docs.google.com/document/d/1cx9xYRlg9LigbDWKL0JsAeGNzjW0-SdWWJM9uPABo-Q/edit).

That's why feedback is welcome at this point. I'll move this one to `shared`. Thanks!

Reply:
Awesome, thanks for confirming!